<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComputerDialog</class>
 <widget class="QDialog" name="addComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_photo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>91</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Photo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_computer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Computer to list</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_Add_Info">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_Input_Name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_Input_Year_Built">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_Input_Development">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="computer_Input_Type">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>251</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_add_computer_photo">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add photo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_add_computer_info">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>200</y>
     <width>44</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Add info:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_Year_Built">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>71</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Year Built :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_Computer_Development">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Development :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_comptuer_name">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>43</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_Type">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>130</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>addComputerDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
